server/models/users.js

```diff
@@ -71,7 +87,36 @@ const getUserModel = (sequelize, Sequelize) => {
       target: "id",
       onUpdate: "CASCADE"
     });
+    User.hasMany(db["Articles"], {
+      foreignKey: "authorId",
+      target: "id",
+      onDelete: "CASCADE"
+    });
+  };
+
+  User.prototype.generateResetToken = function() {
```

Comment:
Why on the prototype and not on the function?. Also I was able to use arrow functions in my helloBooks project. Go an check it out